### ISSUE TITLE
Online developer meeting every other Tuesday at 2:30pm (Eastern time)

### ISSUE THREAD

**Issue by MattHJensen** [PSLmodels/Tax-Calculator#2065]
See the [PSL events calendar](https://www.pslmodels.org/events) for the location and date of the next meeting.

**Comment by MattHJensen** [comment]
I just added another option to the Doodle Poll: 5 pm - 6 pm EST. 

If you already filled out the poll, apologies -- could you please return and indicate if that time works for you?

**Comment by MattHJensen** [comment]
**We are on for 5 pm EST on Thursday, October 11.** 

If you can only make it for part of the call, that is fine, please still join if you are interested. 

Thank you everyone for participating in the Doodle Poll. We can discuss the time slot further on the call. 

Please see the [agenda](https://docs.google.com/document/d/1sFUgYVVx_XzJ3DuOKH_CNhGitSkC6uM2zn5EmpDiT6o/edit?usp=sharing) ahead of time, and feel free to suggest edits. 

LOCATION:

Join Webex meeting
https://ospc.my.webex.com/join/matt.jensen  |  628 792 770

Join by phone
+1-510-338-9438 USA Toll
Access code: 628 792 770
[Global call-in numbers](https://mail.aei.org/owa/redir.aspx?C=3bfxNY8Hq70WhitL40PkEkH-cY-nbyII6k1CpEi38L8HKLh7xg7WCA..&URL=https%3a%2f%2fospc.my.webex.com%2fcmp3300%2fwebcomponents%2fwidget%2fglobalcallin%2fglobalcallin.do%3fsiteurl%3dospc.my%26serviceType%3dMC%26ED%3d712667622%26tollFree%3d0)

**Comment by feenberg** [comment]
I never got a phone number and code - am I not expected at this?

dan

**Comment by hdoupe** [comment]
Hey Dan, I was able to join at : https://ospc.my.webex.com/join/matt.jensen

I'm not sure how to dial in from a phone.

**Comment by MattHJensen** [comment]
@feenberg: 

+1-510-338-9438
628 792 770

**Comment by MattHJensen** [comment]
Hi all, looking forward to seeing everyone available and interested at the semi-monthly tax-calculator developers standing meeting today at 5 pm. 

**Join Webex meeting
https://ospc.my.webex.com/join/matt.jensen | 628 792 770

Join by phone
+1-510-338-9438 USA Toll
Access code: 628 792 770
Global call-in numbers**

**Comment by donboyd5** [comment]
I called in and was on web for about 15 mins but never got in. I can try again if it's going on.

**Comment by donboyd5** [comment]
Will be on the next one, Thurs 11/8 5pm eastern.

**Comment by MaxGhenis** [comment]
@donboyd5 did you try yesterday or today? We met yesterday.

**Comment by donboyd5** [comment]
You outed me, @MaxGhenis. I blew it. I tried today. You'd think I never used a computer or a calendar. It's now on my electronic calendar for every other Thursday, forever.

Responding to your other note in a minute.

**Comment by MaxGhenis** [comment]
NP! Here's a link for anyone who wants to add it to their calendar directly: https://calendar.google.com/event?action=TEMPLATE&tmeid=MHFjMjdzcmJ2MWhzMzRhaW1sMjFnbHJqcWxfMjAxODExMDhUMjIwMDAwWiBtZ2hlbmlzQG0&tmsrc=mghenis%40gmail.com&scp=ALL

I made it every two weeks rather than semi-monthly (i.e., 2nd and 4th Thursdays), which may matter down the line.

**Comment by MaxGhenis** [comment]
In today's meeting we confirmed that this meeting will be every two weeks rather than semi-monthly, so the [calendar event](https://calendar.google.com/event?action=TEMPLATE&tmeid=NmVuOGh2N3RyNTdvbDRhamFwcWxzcGdrcDZfMjAxODExMDhUMjIwMDAwWiBhb2U2Z2QydGo4cGs2amdjYWw4YjNsa29tb0Bn&tmsrc=aoe6gd2tj8pk6jgcal8b3lkomo%40group.calendar.google.com&scp=ALL) is correct.

I deleted the next instance which falls on Thanksgiving. We're on for December 6 next.

**Comment by hdoupe** [comment]
Is the developer meeting today or next week?

**Comment by MaxGhenis** [comment]
On today's call, I asked whether the next meeting is normally scheduled. To follow up: this was probably needless, as it's scheduled for January 3 which seems fine.

**Comment by MaxGhenis** [comment]
For those unsure of whether it's been an even number of weeks since 2018-10-25, you can add [this Google Calendar event](https://calendar.google.com/event?action=TEMPLATE&tmeid=NmVuOGh2N3RyNTdvbDRhamFwcWxzcGdrcDZfMjAxOTA0MjVUMjEwMDAwWiBhb2U2Z2QydGo4cGs2amdjYWw4YjNsa29tb0Bn&tmsrc=aoe6gd2tj8pk6jgcal8b3lkomo%40group.calendar.google.com&scp=ALL) (we're meeting today).

**Comment by martinholmer** [comment]
No online Tax-Calculator meeting on July 4, 2019.
Next meeting is on July 18th.

**Comment by MattHJensen** [comment]
I am unable to make the 5pm time for many of the meetings, and I've heard that it has been difficult to get into the meeting room in my absence. 

Would others be open to rescheduling these meetings to 3p, same days? 

(I am not suggesting that we reschedule today's meeting.)

**Comment by donboyd5** [comment]
Yes, generally I have been unable to make 5 pm on Thursdays. The last two
times that I was able to make it, I tried and was unable to get into the
meeting. I generally should be able to make meetings at 3 pm on Thursdays.

Don

**Comment by codykallen** [comment]
3pm should probably be fine for me.

**Comment by MattHJensen** [comment]
I spoke w @martinholmer and he is fine moving this meeting to 3p. 

Thanks to everyone for being open to this time change. I thought these meetings were extremely productive before they got off track, and I'm excited to be able to attend regularly again and kick the meetings off smoothly. 

The next meeting will be 3p on September 12th.  

cc @Peter-Metz re the PSL catalog update.

**Comment by donboyd5** [comment]
Great. On my calendar.

Don

**Comment by MaxGhenis** [comment]
Updated the [Google Calendar event](https://calendar.google.com/event?action=TEMPLATE&tmeid=M3BmYWsydG80MzVmNzZhYXFmaW5uY3JuNWdfMjAxOTA5MTJUMTkwMDAwWiBhb2U2Z2QydGo4cGs2amdjYWw4YjNsa29tb0Bn&tmsrc=aoe6gd2tj8pk6jgcal8b3lkomo%40group.calendar.google.com&scp=ALL).

Here's the [notes doc](https://docs.google.com/document/d/1sFUgYVVx_XzJ3DuOKH_CNhGitSkC6uM2zn5EmpDiT6o), if anyone wants to suggest agenda items.

**Comment by MaxGhenis** [comment]
Is this still happening? Nobody joined today, nor IIRC two weeks ago.

**Comment by codykallen** [comment]
@MaxGhenis asked:

> Is this still happening? Nobody joined today, nor IIRC two weeks ago.

I have class during these, so I've been missing them most of the time. But if these meetings are at bad times for most people or are unnecessarily frequent, we should consider rescheduling.

**Comment by MattHJensen** [comment]
> Is this still happening? Nobody joined today, nor IIRC two weeks ago.

I'm sorry I missed the meeting this week. I lost track of time on the road today. We did have the meeting two weeks ago, but attendance was light so we ended after 15 min or so. 

I would be open to rescheduling to find a time that is more convenient for Cody and any others who can't make 3p on Thursday. 

2:30 Eastern every other Tuesday is my preference, so we can have 30 min right before the PSL meeting, but I can send out a poll if there isn't unanimous support for that time.
